doc/backend/04http_header.md

```diff
@@ -0,0 +1,9 @@
+# HTTP Headers
+
+This document explains the HTTP headers used by the backend.
+
```

Comment:
Link https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html here.

Reply:
I linked mozilla doc instead. The one you linked is stated to be superseded and mozilla has a more specific one.